ecommerce-ui/src/components/bnbRentalCard/BnbRentalImage.jsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { Box, CardMedia } from "@mui/material";
+import PropTypes from "prop-types";
+
+function BnbRentalImage({ image, imageAlt }) {
+  return (
+    <Box display={"flex"} justifyContent={"center"} alignItems={"center"}>
+      <CardMedia
+        component="img"
+        src={image}
+        alt={imageAlt}
+        sx={{
```

Comment:
I like what you're doing here, but you'll want to keep as much styling into CSS files as you can.

Reply:
Ok got it.  I actually started learning MUI a few weeks ago and saw we could use it in this hw and had fun with it.  Good to know it's ok to include CSS files too.